integrations/kubernetesConfig/cli/init.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+readonly ROOT_DIR="$(dirname "$0")/../../.."
+readonly COMMON_DIR="$ROOT_DIR/integrations/common"
+readonly HELPERS_PATH="$COMMON_DIR/_helpers.sh"
+readonly LOGGER_PATH="$COMMON_DIR/_logger.sh"
+
+# shellcheck source=integrations/common/_helpers.sh
+source "$HELPERS_PATH"
+# shellcheck source=integrations/common/_logger.sh
+source "$LOGGER_PATH"
+
+export KUBERNETES_KUBECONFIGFILE=""
+export KUBERNETES_INT=""
+
+help() {
+  echo "
+  Usage:
+    $SCRIPT_NAME <resource_name> [scopes]
+  "
+}
+
+check_params() {
+  _log_msg "Checking params1"
+
+  KUBERNETES_INT="$(shipctl get_integration_resource "$RESOURCE_NAME")"
+  KUBERNETES_KUBECONFIGFILE="$( echo "$KUBERNETES_INT" | jq -r '.kubeConfigContent' )"
+
+  if _is_empty "$KUBERNETES_KUBECONFIGFILE"; then
+    _log_err "Missing 'kubeConfigFile' value in $RESOURCE_NAME's integration."
+    exit 1
+  fi
+
+  _log_success "Successfully checked params"
+}
+
+init_scope_configure() {
+  _log_msg "Initializing scope configure"
+
+  mkdir /root/.kube
```

Comment:
This is a bit strange. We are saving config in /root, but later we clean up in ~/. I'm assuming this didn't work in node, so a workaround was used, but this should be fixable now right?

Reply:
Isn't saving it in ~/ better?